app/src/webview/java/org/mozilla/focus/webview/SystemWebView.java

```diff
@@ -177,6 +176,15 @@ public void loadUrl(String url) {
         client.notifyCurrentURL(url);
     }
 
+    @Override
+    public void onPause() {
+        // Ensure any videos that are playing get dismissed before we background the application.
+        // https://github.com/mozilla-mobile/firefox-tv/issues/460
+        callback.onExitFullScreen();
```

Comment:
Is there something that needs to be called in onResume, to undo this if we end up coming back to the video without doing something else?

Reply:
@liuche The crash happens if the app gets backgrounded in any way (Just bringing up the voice control and then pressing back will cause it). I'm not sure of other cases `onPause` will get called that we will need to account for?

Reply:
Okay, so bringing up the voice control and dismissing it will return the video to an expected state? If so, that seems fine for handling the onResume bc it seems like it just works.